File: Alura/estudos/javascript007/Cliente.js
export default class Cliente {

    constructor(nome,cpf,senha) {
        this.nome = nome;
        this._cpf = cpf;
        this._senha = senha;
    };

    get cpf() {
        this._cpf;
    };

    autenticar() {
        return true;
    };
};

File: Alura/estudos/kotlin/curso_kotlin_007_funcoes/src/main/kotlin/br/com/alura/bytebank/modelo/Cliente.kt
package br.com.alura.bytebank.modelo

class Cliente(
    var nome: String,
    val cpf: String,
    var endereco: Endereco = Endereco(),
    private val senha: Int
) : Autenticavel {
    override fun autentica(senha: Int): Boolean {
        if (this.senha == senha) {
            return true
        }
        return false
    }

    init {
        println("Criando cliente: ")
        println(this.nome)
        println(this.cpf)

    }

}
